src/Templates/Profile/user.twig

```diff
@@ -155,7 +155,7 @@
             <em>by <a href="{{train.awarded_by.url}}">{{train.awarded_by.value}}</a>
               {% if train.revoked_by %}
                 , revoked on {{train.revoked_time|date("j/n/Y")}} by
-                <a href="{{train.revoked_by.url}}">{{train.revoked_by.fname}} {{train.revoked_by.sname}}</a>
+                <a href="{{train.revoked_by.url}}"> {{train.revoked_by.fname}} {{train.revoked_by.sname}}</a>
```

Comment:
is this a fix for something?

Reply:
This was from when I added name to trainer field however I decided against it as it might confuse fresher